src/main/java/org/benetech/servicenet/config/QuartzConfig.java

```diff
@@ -47,35 +43,22 @@
     @Autowired
     private ApplicationContext applicationContext;
 
+    @Autowired
+    private QuartzProperties quartzProperties;
+
     @Bean
     @Profile({ SPRING_PROFILE_PRODUCTION, SPRING_PROFILE_DEVELOPMENT} )
     public SchedulerFactoryBean schedulerFactoryBean() {
-        SchedulerFactoryBean scheduler = new SchedulerFactoryBean();
+        SchedulerFactoryBean schedulerFactoryBean = new SchedulerFactoryBean();
 
         AutowiringBeanJobFactory jobFactory = new AutowiringBeanJobFactory();
         jobFactory.setApplicationContext(applicationContext);
-        scheduler.setJobFactory(jobFactory);
-
-        List<BaseJob> jobs = new ArrayList<>();
-        jobs.add(edenDataUpdateJob);
```

Comment:
you can remove the autowired jobs if they are not used anymore.

Reply:
fixed